setup.py

```diff
@@ -6,7 +6,7 @@
 with open("README.md") as readme_file:
     readme = readme_file.read()
 
-requirements = ["typer", "requests", "pydantic", "pyyaml", "pydantic", "appdirs"]
+requirements = ["typer", "requests", "pydantic", "pyyaml", "pydantic", "appdirs","networkx"]
```

Comment:
Nitpicking formatting

```suggestion
requirements = ["typer", "requests", "pydantic", "pyyaml", "pydantic", "appdirs", networkx"]
```